src/components/AppHeader/Notifications/index.tsx

```diff
@@ -7,12 +7,12 @@ import { Notifications as UINotifications, useDevice } from '@deriv-com/ui';
 export const Notifications = () => {
     const [isOpen, setIsOpen] = useState(false);
     const { localize } = useTranslations();
-    const { isDesktop } = useDevice();
+    const { isMobile } = useDevice();
```

Comment:
We have to becareful with when we use isDesktop/isMobile as there will be cases when Desktop and Tablet sizing is the same, and sometimes Tablet and Mobile sizing is the same. I checked in the figma and it varies slightly 😢